legal-api/tests/unit/resources/v2/test_business_filings/test_filings.py

```diff
@@ -1028,6 +1028,10 @@ def test_calc_annual_report_date(session, client, jwt):
 CONTINUATION_OUT_FILING = copy.deepcopy(FILING_HEADER)
 CONTINUATION_OUT_FILING['filing']['continuationOut'] = {}
 
+# FUTURE: use AGM_LOCATION_CHANGE_FILING from business schema data when restoration filing work has been done
```

Comment:
"when restoration filing work has been done" should be "when AGM location change filing work has been done"